### DISCUSSION TITLE
Cleaning branches

### DISCUSSION THREAD

**Discussion by valentinMachado** [VCityTeam/UD-Viz#707] [Q&A]
I am cleaning branches but some are not mine.

https://github.com/VCityTeam/UD-Viz/tree/feature-doxygen => we are currently using [jsdoc](https://jsdoc.app/) to generate the documentation, can we delete this branch @EricBoix ?

https://github.com/VCityTeam/UD-Viz/tree/bootstrap => we have decided to not fix an ui framework for ud-viz (it could be something split in another package based on ud-viz), can we delete this branch @CorentinGaut 

https://github.com/VCityTeam/UD-Viz/tree/AvatarCollisionFeature => the automatic collision management has been implemented in the current version of the master (within this [PR](https://github.com/VCityTeam/UD-Viz/pull/542) ), but not the example though, have you planned to finish this example @thibaultdupont-cpe ?

https://github.com/VCityTeam/UD-Viz/tree/draganddrop => what this widget is supposed to do @clementcolin ?

https://github.com/VCityTeam/UD-Viz/tree/TemporalEnhancement => this is outdated with the current 3DTiles style API, can we delete this @CorentinGaut ?

**Comment by clementcolin** [comment]
> https://github.com/VCityTeam/UD-Viz/tree/draganddrop => what this widget is supposed to do @clementcolin ?

This widget should make it possible to drag and drop a 3DTiles tileset from a local file. However, it was done during an UD-Viz sprint and kind of very behind the main branch. It required some changes to Itowns too. Maybe we can archive it and mention it in a issue in Itowns directly since it is not an UD-Viz feature ?

**Reply by valentinMachado** [reply]
Yes it should be something done in itowns, however I think that's not possible since a tileset have to hosted by an http server and cannot be read from a local file. (C3DTilesLayer sends http request to actually load tile [see](https://github.com/iTowns/itowns/blob/cefebce5d5f453dafefc71830cb8b94373430400/src/Process/3dTilesProcessing.js#L9))
